site/_scss/blocks/_card-section.scss

```diff
@@ -0,0 +1,347 @@
+/* stylelint-disable max-nesting-depth*/
+/* stylelint-disable selector-max-compound-selectors*/
+/* TODO - Need to work on removing 2/3 level nesting. */
```

Comment:
Please do not leave todos or unresolved / unjustified linter disablers in this PR, since it is not likely we will come in and fix this code after the project is launched.

Reply:
Removed todos and added proper comment for the linter disbaler.